docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md

```diff
@@ -26,18 +26,19 @@ For Ubuntu 20.04 LTS ('focal'), use the following file:
 wget https://github.com/savonet/liquidsoap/releases/download/v1.4.4/liquidsoap-v1.4.4_1.4.4-ubuntu-focal-amd64-1_amd64.deb
 ```
 
-For Debian 11 ('Bullseye'), use the following file:
+For Debian 11 ('Bullseye'), use the following files: (Debian 11 ships fdk-aac2, so you'll need to download fdk-aac1 from Debian 10)
```

Comment:
This is not a good recommendation, though I'm not sure what the alternative is. For 1 you risk making a [FrankenDebian](https://wiki.debian.org/DontBreakDebian#Don.27t_make_a_FrankenDebian) and 2 you then don't get security updates for this library.

Reply:
It's the same point we have with the main liquidsoap binary. We already have red boxes all over this doc saying it is unsupported, and if it breaks you own both parts. I just stumbled upon this today cause i had to do this on debian proper for the first time. I don't see the real difference between pulling in a 3rd party deb and an oldoldstable one. Both means you are responsible for the result.

Reply:
I added a comment to the caution.